docs/DeveloperGuide.md

```diff
@@ -2,24 +2,150 @@
+
+#### Class Diagram
+![](team/LogInCommand_Class_Diagram_v002.png)
+1.) The LogInCommand checks the UserList to check if the inputted user already exists.
+<br/>
+2.) In the case that the User is a first time user, the LogInCommand will create a new User object then add it into 
+the UserList. 
+
+#### Sequence Diagram
+![](team/LogInCommand_Sequence_v002.png)
+
+1.) When the user inputs a Login Command, a LogInCommand Object is created with the input member Username and Password.
+<br/>
+2.) The Login Command is then executed by sending the users UserList, ui Ui and the User nowUser to the method where 
+it search the users to check if the input Username already exists with the correct password.
+<br/>
+3.) If the User already exists, it will return the User back to the method as currentUser. Otherwise, it will create a
+User with the input Username and Password.
+<br/>
+
+#### Improvements to be made to Version 2.0<br/>
+* In order to protect the User's privacy, the User's Passwords have been encrypted on the save file. 
+* A Cryptography Class with encipherPassword and decipherPassword has been implemented as part of the version 2.0 of 
+WhereGotTime.
 
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+Our Application, WhereGotTime is developed for University students.
 
 ### Value proposition
-
-{Describe the value proposition: what problem does it solve?}
+It was developed to solve the inconvenience students face when accessing timetables and lecture venues and
+to keep track of their timetable.
+It is also developed to assist in finding common available time slots for students to study with their peers.
 
 ## User Stories
 
 |Version| As a ... | I want to ... | So that I can ...|
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+|v1.0|new user|login to my own timetable|be sure that it is my own timetable that is shown there|
+|v1.0|new user|add events into my timetable for the day|refer to the timetable to remind me of the event|
+|v1.0|new user|edit events that I have entered into my timetable for the day|easily change the details as I need it|
+|v1.0|new user|delete events in my timetable for the day|remove an event in the timetable that I do not wish to keep|
+|v1.0|new user|clear my timetable for the day|so that I can remove all the entries I have entered wrongly|
+|v1.0|new user|compare my timetable with my friends|schedule a common time for revision together|
+|v2.0|user|have my password encrypted|only I can access my own timetables|
+|v2.0|user|be assured that my inputs are correctly added|my input are correctly convey into the timetable|
+|v2.0|user|access my saved timetables|I do not have to manually enter the timetables again|
 
 ## Non-Functional Requirements
```

Comment:
This part was  left empty. 
![image](https://user-images.githubusercontent.com/47693437/97457993-3b330700-1975-11eb-917b-835aa5b1aca1.png)